WooCommerce/Classes/ViewRelated/Orders/Order Details/Address Edit/EditAddressForm.swift

```diff
@@ -35,6 +55,17 @@ struct EditAddressForm: View {
     @State var showStateSelector = false
 
     var body: some View {
+        NavigationView {
+            // navigation liks require wrapping in a stack to work
```

Comment:
```suggestion
            // navigation links require wrapping in a stack to work
```

Do you know why this happens? Is it due to a bug? or does it has a logical reason?

Reply:
I don't have definite answer here, I remember using simple links directly inside NavigationView just fine.
Maybe it's because we use `EmptyView` in links.